dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -719,6 +751,13 @@ internal open class RumViewScope(
         internal const val ACTION_DROPPED_WARNING = "RUM Action (%s on %s) was dropped, because" +
             " another action is still active for the same view"
 
+        internal const val RUM_CONTEXT_UPDATE_IGNORED_AT_STOP_VIEW_MESSAGE =
+            "Trying to update global RUM context when StopView event arrived, but context" +
+                " doesn't reference this view already."
```

Comment:
I think instead of `doesn't reference this view already` which is a bit confusing maybe you can say ` that the context doesn't reference this view anymore` or simply `doesn't reference this view`

Reply:
Indeed. I've applied this suggestion.